lace/archive/base_archive.py

```diff
@@ -468,7 +468,7 @@ def plot_3D_samples(self, param_1, param_2, param_3):
 
         return
 
-    def print_entry(self, entry, fiducial_keys=True):
+    def print_entry(self, entry):
```

Comment:
`fiducial_keys` is no longer an argument, it should be removed from the docstring

Reply:
Done!